### DISCUSSION TITLE
bat file missing from the focus folder

### DISCUSSION THREAD

**Discussion by 37OMKAR** [lllyasviel/Fooocus#2722] [Q&A] [answered]
after i restarted my pc the focus launch bat file was missing so i extracted it again from the zip folder and psted it in the foocus win 64 folder but it did not work, i also did a git pull and updated the foocus but still cant find the bat file, can checked my antivirus qurantien log its not there, now i am uanble to use focus

**Comment by 37OMKAR** [comment]
F:\focusw\Fooocus_win64_2-1-831>.\python_embeded\python.exe -s Fooocus\entry_with_update.py
'.\python_embeded\python.exe' is not recognized as an internal or external command,
operable program or batch file.

F:\focusw\Fooocus_win64_2-1-831>pause
Press any key to continue . . .

**Comment by mashb1t** [comment, marked as answer]
Please delete the Fooocus folder and extract everything again from the zip file to ensure a clean installation.

**Reply by 37OMKAR** [reply]
i did that was extracted run.bat from the zip, then run it as admin, it worked, 
just keep it in folder 



F:\Fooocus_win64_2-1-831>.\python_embeded\python.exe -s Fooocus\entry_with_update.py
Already up-to-date
Update succeeded.
[System ARGV] ['Fooocus\\entry_with_update.py']
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
Fooocus version: 2.3.1
[Cleanup] Attempting to delete content of temp dir C:\Users\Omkar\AppData\Local\Temp\fooocus
[Cleanup] Cleanup successful
Total VRAM 12288 MB, total RAM 57262 MB
Set vram state to: NORMAL_VRAM
Always offload VRAM
Device: cuda:0 NVIDIA GeForce RTX 2060 : native
VAE dtype: torch.float32
Using pytorch cross attention
Refiner unloaded.
Running on local URL:  http://127.0.0.1:7865

To create a public link, set `share=True` in `launch()`.
model_type EPS
UNet ADM Dimension 2816
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE
extra {'cond_stage_model.clip_l.text_projection', 'cond_stage_model.clip_l.logit_scale', 'cond_stage_model.clip_g.transformer.text_model.embeddings.position_ids'}
Base model loaded: F:\Fooocus_win64_2-1-831\Fooocus\models\checkpoints\juggernautXL_v8Rundiffusion.safetensors
Request to load LoRAs [['sd_xl_offset_example-lora_1.0.safetensors', 0.1], ['None', 1.0], ['None', 1.0], ['None', 1.0], ['None', 1.0]] for model [F:\Fooocus_win64_2-1-831\Fooocus\models\checkpoints\juggernautXL_v8Rundiffusion.safetensors].
Loaded LoRA [F:\Fooocus_win64_2-1-831\Fooocus\models\loras\sd_xl_offset_example-lora_1.0.safetensors] for UNet [F:\Fooocus_win64_2-1-831\Fooocus\models\checkpoints\juggernautXL_v8Rundiffusion.safetensors] with 788 keys at weight 0.1.
Fooocus V2 Expansion: Vocab with 642 words.
Fooocus Expansion engine loaded for cuda:0, use_fp16 = True.
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
[Fooocus Model Management] Moving model(s) has taken 3.21 seconds
Started worker with PID 26316
App started successful. Use the app with http://127.0.0.1:7865/ or 127.0.0.1:7865
1006
1006
1006
1006
1006
1006
1006

**Reply by mashb1t** [reply]
=> see https://github.com/lllyasviel/Fooocus/issues/2723

**Reply by JADYMIR45** [reply]
I did everything but it's still very slow to create the images, any suggestions please
